docs/en_US/install/3ds.md

```diff
@@ -21,12 +28,22 @@ Nimbus is available as both a 3DSX app and an installable CIA. The releases page
 
 Extract the contents of the zip archive to the root of your SD card. If you are asked to merge or overwrite files, accept the changes.
 
-You should now have the `0004013000002F02`, `0004013000003202`, and `0004013000003802` Luma patches along with the `nimbus.3dsx` and `nimbus.cia` homebrew.
+Ensure your SD card has all the following files
+
+- `SD:/luma/titles/000400300000BC02/code.ips` (Miiverse, JPN)
+- `SD:/luma/titles/000400300000BD02/code.ips` (Miiverse, USA)
+- `SD:/luma/titles/000400300000BE02/code.ips` (Miiverse, EUR)
+- `SD:/luma/sysmodules/0004013000002F02.ips` (SSL)
+- `SD:/luma/sysmodules/0004013000003202.ips` (Friends List)
```

Comment:
Sounds good. I labeled it this way because that's how it's labeled on 3dbrew, and I wanted to be sure people knew what it actually did. But you're right that this is inaccurate naming as it is

How about `FRD (Friends)`? To cover all bases?

Reply:
Sounds good